src/docs/forms/components/TextInput/examples/desc/02-SimpleTextInput.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+export default class SimpleTextInput extends React.Component {
```

Comment:
For consistency, rename the class using the same class name of the example this points to. Consequently, use the same filename of the example this description points to. For example. if the example class was renamed to `PasswordInput` with file name `02-PasswordInput.js`:

1. Rename the description file to `02-PasswordInput.js`.
2. Rename the description class to `PasswordInput`

Also do this to other examples.